lib/hexdocs/search/typesense.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Hexdocs.Search.Typesense do
+  require Logger
+
+  @behaviour Hexdocs.Search
+
+  @collection "hexdocs"
```

Comment:
random: I tried changing this string to "hexdocsv2" and I started getting 404s when indexing, is this hardcoded somewhere else?

Reply:
I made the collection name configurable in https://github.com/hexpm/hexdocs/pull/44/commits/6116a98a8b3c351bdd4e46c44bdba76863a4a31e, it used to be hardcoded in tests.